### ISSUE TITLE
Would it be posible to show typed value for x time before obscuring text?

### ISSUE THREAD

**Issue by nombrekeff** [Tkko/Flutter_PinPut#68]
Hey there!

I was wondering if it would be possible to show the typed value for some amount of time before obscuring it? Similar to how regular TextFields handle obscured text.

**Current behavior:**
1. PinPut with obscureText = '*'
2. user types character
3. character is obscured

**Requested behavior:**
1. PinPut with obscureText = '*'
2. user types a character
3. character is shown for 1s 
4. Then it obscures itself

I think this could be a really good feature to have, maybe opt-in. This way the user can see if an incorrect character has been entered.

I wouldn't mind filing a PR if the feature is accepted, just let me know! 

Cheers

**Comment by mathiasgodwin** [comment]
Has this been implemented yet?

**Comment by Tkko** [comment]
@mathiasgodwin Not yet 🤭 , PR's are welcomed

**Comment by nombrekeff** [comment]
Yup, that's the case. Any idea on how we could implement this without much work? Or is it even possible with the current implementation? Could we change EditableText to TextFormField or TextField?

**Comment by hungnp-goline** [comment]
Has this feature been implemented yet?
